django/db/models/fields/__init__.py

```diff
@@ -1500,6 +1500,8 @@ def __init__(self, verbose_name=None, name=None, max_digits=None,
                  decimal_places=None, **kwargs):
         self.max_digits, self.decimal_places = max_digits, decimal_places
         super(DecimalField, self).__init__(verbose_name, name, **kwargs)
+        self.validators.append(validators.DecimalValidator(self.max_digits,
+                                                           self.decimal_places))
```

Comment:
You don't need to break this line, if it's below <=119 characters. It's better readable in my eyes.